### PR TITLE
feat: Use Goerli for dev config

### PR DESCRIPTION
**Pull request by fzavalia**

### CODE REVIEW

**Review by LautaroPetaccio — APPROVED** [review]

File: src/config/env/dev.json [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
   "SHARE_SCENE_URL": "https://share.decentraland.zone/b",
   "INTERCOM_APP_ID": "z0h94kay",
   "SEGMENT_API_KEY": "H21EgRI4eYwICDZf5uW6ek2BiykIR6wA",
   "ROLLBAR_ACCESS_TOKEN": "46e4b7e45c844b9ab81315c8b0919e99",
   "ENVIRONMENT": "development",
-  "MANA_TOKEN_CONTRACT_ADDRESS": "0x2a8fd99c19271f4f04b1b7b9c4f7cf264b626edb",
-  "LAND_REGISTRY_CONTRACT_ADDRESS": "0x7a73483784ab79257bb11b96fd62a2c3ae4fb75b",
-  "ESTATE_REGISTRY_CONTRACT_ADDRESS": "0x124bf28a423b2ca80b3846c3aa0eb944fe7ebb95",
-  "ERC721_COLLECTION_FACTORY_CONTRACT_ADDRESS": "0x16d8bac5b67a6b782a9081377bec413bc5bb56a6",
-  "ERC721_COLLECTION_CONTRACT_ADDRESS": "0xc3c8fb76108673f546e8c97d168af61c5573480b",
```
Comment (nicosantangelo) [inline src/config/env/dev.json:20]: I think this address is still being used in `contracts.ts`:

```ts
export const ERC721_COLLECTION_ADDRESS = config.get('ERC721_COLLECTION_CONTRACT_ADDRESS', '')
```

shouldn't we delete it from there too?

**Review by nicosantangelo — COMMENTED** [review]
Question

File: src/config/env/dev.json (hunk 1 above)
Comment (fzavalia) [reply src/config/env/dev.json:20]: image

**Review by nicosantangelo — APPROVED** [review]

**Comment by nicosantangelo** [comment]
While we're at it, shouldn't we delete the vars from `prod.json` and `stg.json`

**Pull request merged**